appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java

```diff
@@ -91,7 +93,8 @@
     category = ComponentCategory.ANIMATION)
 @SimpleObject
 @UsesPermissions(permissionNames = "android.permission.INTERNET," +
-                 "android.permission.WRITE_EXTERNAL_STORAGE")
+                 "android.permission.WRITE_EXTERNAL_STORAGE," +
+                 "android.permission.CAMERA" )
```

Comment:
@thequixotic Why do you add CAMERA permission for this change?

(Sorry for the slow response, we are getting ready to do a Component release [first in a long time] and I want to get this change).